target/linux/ramips/dts/mt7621_tplink_ec330-g5u-v1.dts

```diff
@@ -0,0 +1,293 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "tplink,ec330-g5u-v1", "mediatek,mt7621-soc";
+	model = "TP-Link EC330-G5u v1";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led-0 {
+			label = "blue:usb";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_STATUS;
```

Comment:
I recommend using LED_FUNCTION_USB instead, and the label can be removed. You can find a lot of LED functions here https://github.com/torvalds/linux/blob/v5.10/include/dt-bindings/leds/common.h.

Reply:
Thanks for the piece of advice! I'll push the changes soon.

Reply:
Leds was adjusted,